cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -733,4 +733,29 @@ public static int caCertGeneration(Ca ca) {
     public static int idOfPod(String podName)  {
         return Integer.parseInt(podName.substring(podName.lastIndexOf("-") + 1));
     }
+
+    /**
+     * Generates all possible DNS names for a Kubernetes service:
+     *     - service-name
+     *     - service-name.namespace
+     *     - service-name.namespace.svc
+     *     - service-name.namespace.svc.dns.suffix
```

Comment:
I did not wanted to use the `<...>` format anywhere because you cannot use it directly in JavaDocs. You need to escape it as `&lt;` and `&gt;`. So it it is hard to make it compile but also readable as JavaDoc in the IDE. That is why I didn't used it for the DNS suffix, service name or namespace.